popup/src/components/TimeCheckForm.js

```diff
@@ -27,8 +30,8 @@ export default function TimeCheckForm() {
         onChange={handleTimeChoice}
         type="number"
         name="playAtSecond"
-        label="Play when x seconds left"
-        defaultValue={playAtSecond}
+        label={`Play when ${playAtSecond} seconds left`}
+        value={playAtSecond}
```

Comment:
did this have to be changed? does it work still if you reverse this change

Reply:
Yes. defaultvalue was not an acceptable prop (https://v3.mui.com/api/text-field/#props).

Reply:
Yes, but it wasn't updating with the hooks. The console was registering the change, but the value wouldn't update because it would only use the defaultValue (which was set at 20).

Reply:
OHhh wow